Sources/dep/Git.swift

```diff
@@ -43,9 +43,8 @@ class Git {
         }(self)
 
         var versions: [Version] {
-            //TODO separator is probably \r\n on Windows
             let out = (try? popen([Git.tool, "-C", root, "tag", "-l"])) ?? ""
-            let tags = out.characters.split("\n")
+            let tags = out.characters.split { $0 == "\n" || $0 == "\r\n" }.map(String.init)
```

Comment:
The behavior of `characters` is a little funky:
<img width="300" alt="screen shot 2015-12-09 at 10 43 52 am" src="https://cloud.githubusercontent.com/assets/58962/11694982/bfdafda8-9e61-11e5-9fa5-45b8c3d14616.png">